docs/bugs/null_no_tests_found.md

```diff
@@ -0,0 +1,114 @@
+# The problem
+One or more shards failed because there are no test cases inside. This problem encounter a couple times on submodule tests which use a lot of additional test apks. It was noticed only with the submodule tests.
```

Comment:
submodule means something different: https://git-scm.com/book/en/v2/Git-Tools-Submodules

I think you mean multi-module apps.

Reply:
Right, that is slightly redacted copy paste from #818 description. I will fix.